src/client/app/effects/github.ts

```diff
@@ -134,12 +141,17 @@ export class GitHubEffects {
                 new Snippet.UpdateInfoAction({ id: snippetId, gist: gist.id, gistOwnerId: gist.owner.login })])
         )
         .catch(exception => {
+            let { gistOwnerId, id } = exception;
```

Comment:
Will there ever be another exception that could get thrown here, that isn't your exception?  Should you do some sort of an instanceof check?  (Yes, you should; this ties in to the other comment above)